src/main/java/org/commcare/formplayer/db/migration/V22__create_form_definition.java

```diff
@@ -0,0 +1,26 @@
+package org.commcare.formplayer.db.migration;
+
+import java.util.Arrays;
+
+public class V22__create_form_definition extends BaseFormplayerMigration {
+    @Override
+    public Iterable<String> getSqlStatements() {
+        return Arrays.asList(
+                "CREATE TABLE form_definition (\n" +
+                        "    id text PRIMARY KEY,\n" +
```

Comment:
just looking at this again now I'm wondering why this needs to be a UUID. I don't think it matters too much though integer primary keys are marginally more performant. If we do stick with a UUID then this column type should be UUID and not `text` and the Java field should also be a UUID and not a `string` which would also simplify the mapping slightly:

```java
    // if we stick with a UUID then the mapping can be as follows:
    @Id
    @GeneratedValue
    private UUID id;
```

Reply:
Just commenting to say I agree with Simon

Reply:
See cc72e727ce137f742b6c4743acf2325509059aa9